docs/build-on-linea/linea-sdk/index.md

```diff
@@ -27,8 +27,45 @@ The SDK focuses on interacting with smart contracts on both Ethereum and Linea n
 
  5. Claiming messages (use one of the get message methods to grab all the parameters values)
 
-
-The code example below demonstrates how to initialize the SDK with the relevant configuration options, and how to call the featured methods.
+ 
+:::info[New Features]
+
+ The updated Linea SDK package enhances the L1 contract interaction and adds support for the new L1 claiming system, which is based on a Merkle tree and requires a Merkle proof for claiming.
+ 
+ There are three important things to note:
+  - The previous L1 claiming and all functions associated are still supported.
```

Comment:
If it's still supported, wouldn't it be better described as "existing" rather than "previous"? 

Also, I think we're missing a noun after "claiming": claiming process, claiming flow, claiming mechanism, etc.